docs/Changelog.md

```diff
@@ -4739,7 +4739,7 @@ This version of the operator has been available since version 2 of the default O
 
 <dl>
 <dt><tt>axis</tt> : int (default is 0)</dt>
-<dd>Which axis to split on. A negative value means counting dimensions from the back. Accepted range is [-rank, rank-1].</dd>
```

Comment:
> A negative value means counting dimensions from the back. Accepted range is [-rank, rank-1]. [](start = 27, length = 93)

If the old description allows negative axis, the old signature should still allow it.